docs/tempo/website/grafana-agent/_index.md

```diff
@@ -0,0 +1,126 @@
+It sends traces in OTLP format via gRPC.
+
+On top of receiving and exporting traces, the Grafana Agent contains many 
+features that make your distributed tracing system more robust, and 
+leverages all the data that is processed in the pipeline.
+
+## Architecture
+
+The Grafana Agent can run a set of Tempo instances, each of which configures its own tracing pipeline.
+Each pipeline is built using OpenTelemetry,
+and it's architecture mirrors that of the OTel Collector's [design](https://github.com/open-telemetry/opentelemetry-collector/blob/846b971758c92b833a9efaf742ec5b3e2fbd0c89/docs/design.md).
+See the [configuration reference](https://github.com/grafana/agent/blob/main/docs/configuration-reference.md#tempo_instance_config) for all available config options. 
+For a quick start, refer to this [blog post](https://grafana.com/blog/2020/11/17/tracing-with-the-grafana-agent-and-grafana-tempo/).
+
+<p align="center"><img src="https://raw.githubusercontent.com/open-telemetry/opentelemetry-collector/846b971758c92b833a9efaf742ec5b3e2fbd0c89/docs/images/design-pipelines.png" alt="Tracing pipeline architecture"></p>
+
+Having multiple Tempo instances allows you to configure multiple distinct 
+pipelines, each of which collects separate spans and sends them to different 
+backends.
+
+### Receiving traces
+
+The Grafana Agent supports receiving traces in multiple different formats: 
+OTLP (OpenTelemetry), Jaeger, Zipkin, OpenCensus and Kafka.
+
+Each Tempo instance can be configured to receive traces in all these formats.
+Traces that arrive to a Tempo instance will enter the same tracing 
+pipeline, and will be processed equally.
+
+### Pipeline processing
+
+The Grafana Agent processes tracing data as it flows through the pipeline to make the distributed tracing system more reliable and leverage the data for other purposes.
+Such as trace discovery, tail-based sampling, and generating metrics.
+
+#### Batch
+
+The Agent supports batching of traces.
+Batching helps better compress the data and reduce the number of outgoing connections required to transmit the data.
+To configure it, refer to the `batch` block in the [config reference](https://github.com/grafana/agent/blob/main/docs/configuration-reference.md#tempo_instance_config).
+
+#### Attributes manipulation
+
+The Grafana Agent allows for general manipulation of tags on spans that pass through this agent.
+A common use may be to add an environment or cluster variable.
+To configure it, refer to the `attributes` block in the [config reference](https://github.com/grafana/agent/blob/main/docs/configuration-reference.md#tempo_instance_config).
+
+#### Attaching metadata with Prometheus Service Discovery
+
+Prometheus Service Discovery mechanisms enable you to attach the same metadata to your traces as your metrics.
+For example, for Kubernetes users this means that you can dynamically attach metadata for namespace, pod, and container name of the container sending spans.
```

Comment:
`metadata for namespace, pod, and container name of the container sending spans.`
to
`metadata for namespace, pod, and name of the container sending spans.`